website/client/app.vue

```diff
@@ -370,6 +377,19 @@ export default {
     if (loadingScreen) document.body.removeChild(loadingScreen);
   },
   methods: {
+    checkForBannedUser (error) {
+      const AUTH_SETTINGS = localStorage.getItem('habit-mobile-settings');
```

Comment:
looks good, just one thing: why are we parsing the auth settings again if these are stored into `store.state.credentials`? Because they cannot login at all when banned so the credentials are never stored?

Reply:
Hmmm. I figured a user would be banned while logged in but I'm sure both cases would happen. For testing locally, I banned my user while he was logged in.